header.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * The header for our theme
+ *
+ * This is the template that displays all of the <head> section and everything up until <div id="content">
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ *
+ * @package Sterling
+ */
+
+?>
+<!doctype html>
+<html <?php language_attributes(); ?>>
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="profile" href="http://gmpg.org/xfn/11">
+
+	<?php wp_head(); ?>
+</head>
+
+<body <?php body_class(); ?>>
+<div id="page" class="site">
+	<a class="skip-link screen-reader-text" href="#content"><?php esc_html_e( 'Skip to content', 'sterling' ); ?></a>
+
+	<header id="masthead" class="site-header">
+            
+            <div id="search-menu">
+                
+                <div id="search-menu-box">
+                   
+                    <div id="search-icon">
+                        
+                        <i class="fa fa-search fa-2x"></i>
+                        
+                    </div>
+                    
+                    <input type="text" placeholder="Search">
+                    
+                    <div id="close-search-btn">
+                        
+                        <i class="fa fa-times fa-2x"></i>
+                        
+                    </div>
+                    
+                </div>
+                
+            </div>
+            
+            <div id="top-bar" class="container-fluid">
+                
+                <div class="row">
+            
+                    <div id="site-branding" class="col-sm-3">
+                            <?php
```

Comment:
when logo is added, site title & description remain visible. hide title/desc when logo is displayed